p2p/simulations/adapters/exec.go

```diff
@@ -201,7 +201,7 @@ func (n *ExecNode) Start(snapshots map[string][]byte) (err error) {
 	go func() {
 		s := bufio.NewScanner(stderrR)
 		for s.Scan() {
-			if strings.Contains(s.Text(), "WebSocket endpoint opened:") {
+			if strings.Contains(s.Text(), "WebSocket endpoint opened") {
```

Comment:
Someone along the merges broke the logger, and at the moment we are using the default logger. The default logger does not add a colon at the end of log messages, so I am relaxing a bit this check. We are still detecting the websocket addr correctly, which is what matters here.